packages/runtime/src/runtime.ts

```diff
@@ -259,6 +281,36 @@ export class Runtime {
     }
   }
 
+  /**
+   * Creates Asset in Runtime
+   * @param name ASA name
+   * @param flags ASA Deployment Flags
+   */
+  createAsset (name: string, flags: ASADeploymentFlags): number {
```

Comment:
Let's keep the naming similar to `app`. We are using `addApp`, so please rename this to `addAsset`, OR rename `addApp` to `createApp` as well. I would prefer (i think robert suggested this earlier): 
```
runtime.createAsset(..) => account.addAsset(..)
runtime.createApp(..) => account.addApp(..)
```

Reply:
ok